staging/src/k8s.io/client-go/applyconfigurations/OWNERS

```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- apelisse
```

Comment:
Probably should add at least @jpbetz here too if we're creating this file?

Reply:
Thanks, yes, I should be here.